consensus/poet/core/sawtooth_poet/poet_consensus/poet_settings_view.py

```diff
@@ -249,7 +249,7 @@ def signup_commit_maximum_delay(self):
                 self._get_config_setting(
                     name='sawtooth.poet.signup_commit_maximum_delay',
                     value_type=int,
-                    default_value=PoetConfigView._SIGNUP_COMMIT_MAXIMUM_DELAY_,
+                    default_value=self._SIGNUP_COMMIT_MAXIMUM_DELAY_,
```

Comment:
Aside from maybe preventing lint-ing error with respect to 79-column limit, why are some of these `PoetSettingsView` and some are `self`?

I would recommend change all to `self` or change none.